cmake/apple/readme.txt

```diff
@@ -5,12 +5,16 @@ Solarus is licensed under GNU General Public License v3.
 This OSX package contains:
 - Solarus (Solarus.app): A GUI that allows you to run a Solarus quest and
   choose your audio and video settings.
-- Solarus-run (Solarus-run.app): An executable file that directly runs
-  a Solarus quest.
-  Run it with the path of your quest as argument
-  (the directory containing data/ or data.solarus or data.solarus.zip).
+- Solarus-run (Solarus-run.app): A package you can use either to distribute a 
+  standalone quest on other OSX systems with non-developer environment, or to 
```

Comment:
I don't understand why we talk about distributing quests in this readme, or about "non-developer environment". We should simply explain that solarus-run runs a quest.